bundles/org.openhab.binding.speedtest/README.md

```diff
@@ -0,0 +1,92 @@
+
+
+## Why Ookla's Speedtest?
+
+Fully supported and maintained by Ookla with the latest Speedtest technology for best results:
+
+* Consistent measurement, even on high bandwidth
+* Consistent measurement, very much independent from the performance of the host system
+
+
+## What functionality does Ookla's Speedtest offer?
+
+* Output of a timestamp instead of using the openHAB timestamp
+* Output of Ping time and Jitter
+* Output of Bandwidth, transferred Bytes and elapsed time for Down-/Upload
+* Output of the used interface with Internal/External IP, MAC Address and the Internet Service Provider (ISP)
+* Output of a result ID and a result URL
+* Output of the used Server and location
+* Possiblity to pre-select a server used for testing
+
+
+## What functionality does the Speedtest Binding offer?
+
+* Execute Speedtest time based or triggered
+* Provide results via openHAB Channels
+* List available Ookla Speedtest servers that can be used for testing (optional)
+
+
+## Ookla License and Privacy Terms
+
+When using this binding, you automatically accept the license and privacy terms of Ookla's Speedtest.
+You can find the latest version of those terms at the following webpages:
+
+* https://www.speedtest.net/about/eula
+* https://www.speedtest.net/about/terms
+* https://www.speedtest.net/about/privacy
+
+
+## Supported Things
+
+Speedtest thing.
+
+## Binding Configuration
+
+For this binding to work, you MUST install Ookla's Speedtest command line tool (`speedtest` or `speedtest.exe`).
+It will not work with other versions like `speedtest-cli` or other `speedtest` variants.
+
+To install Ookla's version of Speedtest, head to https://www.speedtest.net/apps/cli and follow the instructions for your Operating System.
+
+## Thing Configuration
+
+| Parameter         |  Description                                                                                                                 | Default |
+|-------------------|------------------------------------------------------------------------------------------------------------------------------|---------|
+| `refreshInterval` | How often to test network speed, in minutes                                                                                  | `60`    |
+| `execPath`        | The path of the Ookla Speedtest executable.<br/>Linux machines may leave this blank and it defaults to `/usr/bin/speedtest`. |         |
+| `serverID`        | Optional: A specific server that shall be used for testing. You can pick the server ID from the "Thing Properties".<br/>If this is left blank the best option will be selected by Ookla.          |         |
+
+The `refreshInterval` parameter can also be set to "Do not test automatically".
+This can be used if you want to use the "Trigger Test" channel in order to test via rules, or an item instead.
+
+Ensure that the user that openHAB is running with, has the permissions to access and execute the executable.
+
```

Comment:
I think it would be nice to add a demo.things example. with both bridge and thing configuration in textual configuration.

Reply:
done

Reply:
at the end of the readme, you can find the thing and item file:
https://github.com/openhab/openhab-addons/pull/9913/commits/6ee0a007e28f3a8325addcccd2eb0963769069d5

Reply:
Ah, sorry i overlooked somehow. It looks like the item example is not properly aligned. Thre rest looks good.

Reply:
Fixed the alignment in the readme (tabs -> blanks)